File: cpp-programs/functions/fibonacci.cpp
#include<string.h>
#include<bits/stdc++.h>
using namespace std;

int fibo(int);

int main()
{

long n,t;

cout<<"Enter the term number : ";
cin>>n;

t = fibo(n);

cout<<n<<"th term of fibonacci series is = "<<t<<endl;

}

int fibo(int n)
{
 if(n==0 || n==1)
  return n;
 else
  return fibo(n-1) + fibo(n-2);
}

File: practice-code-blue/programme-python/fibonacci.py
#!/usr/bin/python

def fibonacci(n):
  if(n==0):
     return 0
  if(n==1):
     return 1
  return fibonacci(n-1) + fibonacci(n-2)
  
i=1
while(i<=10):
 print("Fibonacci number ", i ,":", fibonacci(i))
 i= i + 1
